styles/terminal.less

```diff
@@ -71,6 +71,6 @@
 
 atomic-terminal {
   .terminal.xterm {
-    padding: 1rem;
+    padding-left: @component-padding;
```

Comment:
I don't think we should do padding only for one direction. 

```suggestion
    padding: @component-padding;
```

Reply:
the other directions are already padded as I showed

Reply:
again this is consistent with tree-view and other panes

Reply:
Then, we need padding for all directions except right.

```suggestion
    padding: @component-padding;
    padding-right: 0px;
```

Reply:
tree-view doesn't have padding on the top or bottom. There is already padding there on xterm because it only renders full lines

Reply:
That is a lot of bottom padding

![Animation2](https://user-images.githubusercontent.com/97994/125384060-f9f55280-e35d-11eb-88ca-83b9db9614e1.gif)

Reply:
```suggestion
    padding-left: @component-padding;
    padding-top: @component-padding;
```

There is no padding on the top:

![image](https://user-images.githubusercontent.com/16418197/125384479-99b2e080-e35e-11eb-862e-318befc89434.png)

Reply:
vscode also doesn't have any padding on the bottom

![image](https://user-images.githubusercontent.com/97994/125384714-f1514c00-e35e-11eb-8215-219efd7e4aac.png)

They do something weird where the bottom always has no padding and the top padding changes based on the height of the terminal.